lib/ui/semantics.dart

```diff
@@ -348,6 +349,14 @@ class SemanticsFlag {
   ///   * [SemanticsFlag.hasCheckedState], which enables a checked state.
   static const SemanticsFlag isChecked = SemanticsFlag._(_kIsCheckedIndex);
 
+  /// Whether a tristate checkbox is in its mixed state.
+  ///
+  /// Should only be true when the checkbox has [tristate] = true.
```

Comment:
this can still be false if [tristate] = true?

Also this comment is too specific, people who reading this doc may not know what tristate is.

Maybe
```
If this is true, the check box this semantics node represents is in a mixed state. For example, a [Checkbox] with [Checkbox.tristate] set to true can have checked,  unchecked, or mixed state.
```